internal/trivy/scan_job.go

```diff
@@ -73,6 +73,9 @@ func (f *filesystemScanJobBuilder) ForCIS(cis *stasv1alpha1.ContainerImageScan)
 		stasv1alpha1.LabelK8SAppManagedBy:             stasv1alpha1.AppNameImageScanner,
 		stasv1alpha1.LabelStatnettControllerNamespace: cis.Namespace,
 		stasv1alpha1.LabelStatnettControllerUID:       string(cis.UID),
+		stasv1alpha1.LabelStatnettWorkloadKind:        string(cis.Spec.Workload.Kind),
+		stasv1alpha1.LabelStatnettWorkloadName:        string(cis.Spec.Workload.Name),
```

Comment:
I think the name must be truncated. Max label value length is 63 characters, ref. https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set. You can use the constant `DNS1123LabelMaxLength` already in use for the job name max length.